core/src/main/codegen/templates/Parser.jj

```diff
@@ -3980,7 +3968,7 @@ SqlNode Expression3(ExprContext exprContext) :
     {
         if (list1.size() == 1) {
             // interpret as single value or query
-            return list1.get(0);
+            return list1.get(0).clone(list1.getParserPosition());
```

Comment:
Is clone necessary here?

Reply:
this is actually the line that fixes the JIRA bug

Reply:
I understand what you mean